apim/3.x/values.yaml

```diff
@@ -1168,7 +1168,9 @@ ui:
     runAsNonRoot: true
   ingress:
     enabled: true
-    path: /console(/|$)(.*)
+    basePath: /console
```

Comment:
changing path to basePath will be a breaking change. If some users already have a path defined, it will no longer be taken into account.